src/main/java/com/petros/bibernate/dao/EntityPersister.java

```diff
@@ -126,19 +132,13 @@ private static <T> T mapResultSetToEntity(Class<T> entityClass, ResultSet result
             T entity = entityClass.getConstructor().newInstance();
             for (var entityField : entityClass.getDeclaredFields()) {
                 entityField.setAccessible(TRUE);
-                String columnName = getColumnName(entityField);
-                entityField.set(entity, resultSet.getObject(columnName));
+                String columnName = EntityUtil.getColumnName(entityField);
+                Object columnValue = resultSet.getObject(columnName);
+                log.trace("Setting DB->Object, class = {}, field = {}, value = {}", entityClass.getSimpleName(), columnName, columnValue);
+                entityField.set(entity, columnValue);
             }
             return entity;
-        } catch (InstantiationException e) {
-            throw new BibernateException(e);
-        } catch (IllegalAccessException e) {
-            throw new BibernateException(e);
-        } catch (InvocationTargetException e) {
-            throw new BibernateException(e);
-        } catch (NoSuchMethodException e) {
-            throw new BibernateException(e);
-        } catch (SQLException e) {
+        } catch (Exception e) {
```

Comment:
It's generally not a good practice to catch a broad Exception class because it can mask errors and make it difficult to diagnose issues.Consider consolidating these catch blocks into one block that catches all the listed exceptions or their common superclass. Alternatively, you could concatenate the exceptions by using the pipe symbol '|' in the catch block, like this: } catch (InstantiationException | IllegalAccessException | InvocationTargetException | NoSuchMethodException | SQLException | Exception e) {."